docs/implementation-guide.md

```diff
@@ -0,0 +1,122 @@
+
+If you do not have meet either of these criteria contact the owner of your organization's GCP environment.
+
+### Step 2: Install the Gcloud CLI.
+The gcloud CLI is a command line tool provided by Google for working with GCP. We will use it to authenticate your local machine with your organization's GCP environment. Follow [this guide](https://cloud.google.com/sdk/docs/install) to install gcloud. After installation run `gcloud auth application-default login` and follow the prompts in your browser to login to GCP.
+
+### Step 3: Fork the phdi-google-cloud Respository
+Fork the phdi-google-cloud respository into your organization's, or your personal, GitHub account.
+1. Navigate to [https://github.com/CDCgov/phdi-google-cloud](https://github.com/CDCgov/phdi-google-cloud).
+2. Click on the `Fork` button in the top right.
+![fork-repo-1](./fork-repo-1.png)
+3. Select the owner of the forked repository. We recommend that this is an organization and not an individual.
+4. Click `Create fork` at the bottom of the page.
+![fork-repo-2](./fork-repo-2.png)
+
+### Step 4: Clone the Forked Repository
+Clone the forked version of the phdi-google-cloud repository by running `git clone https://github.com/<MY-GITHUB-ORGANIZATION>/phdi-google-cloud.git`. If you do not have `git` installed please follow [this guide](https://github.com/git-guides/install-git) to install it.
+
+### Step 5: Run the Quick Start Script
+In this step we will work through GCP's [Workload Identity Federation](https://cloud.google.com/iam/docs/workload-identity-federation) to granted your phdi-google-cloud repo access to deploy the pipelines to your organization's GCP environment. We have provided a script to automate most of this process that we recommend you use. However if you prefer to work through it manually you may follow [this guide](https://github.com/google-github-actions/auth#setup).
+
+From your machine's command line:
+1. Navigate to the root directory of the repository you cloned in step 4.
+2. Login into GCP by running `gcloud auth application-default login`.
+3. Run the quick start script and follow the prompts.
+    - [quick-start.sh](../quick-start.sh) for Mac an Linux
+    - TODO: WRITE WINDOWS BATCH QUICK START SCRIPT
+
+If you plan to deploy to an existing project in your GCP environment, have the project ID ready and provided it to the quick start script when prompted.
+
+### Step 6: Set Repository Secrets
+Set the following secret values in your forked phdi-google-cloud repository:
+- `PROJECT_ID` - Specified by the quick start script.
+- `SERVICE_ACCOUNT_ID` - Specified by the quick start script.
+- `WORKLOAD_IDENTITY_PROVIDER` - Specified by the quick start script.
+- `REGION` - Your choice of GCP region.
+- `ZONE`- Your choice of GCP zone.
+
+Information about GCP regions and zones is available [here](https://cloud.google.com/compute/docs/regions-zones).
```

Comment:
Let's provide a default in the cli prompt - my preference would be `us-central1` since we're assuming that we'll only have US users and it's "in the middle".

Reply:
Currently the script suggests `us-west1`, but we can change that to `us-central1`. We will have additional work here once we start implementing geo-redundant storage because doing so properly will place additional constrains on choice of region. It will also require specifying multiple zones or regions.

Reply:
I chose us-west1 as the default because it is one of GCP's "low CO2" regions, but apparently so is us-central1, so I'm fine with changing that!

Reply:
The quick start script now suggest `us-central1` for the region and `us-central1-a` for the zone.

Reply:
Oh nice! I picked `us-central1` because it seemed "central" and perhaps best of overall latency. 

Could we explicitly state in the guide what these values default to? 

Something like...
```
`REGION` - Your choice of GCP region. (Defaults to us-central1)
`ZONE`- Your choice of GCP zone. (Defaults to us-central1-a)
```

It's not clear in the guide that a default value will be provided.

Reply:
I don't mean to be pedantic, but because the entire process for creating the repository secrets is currently manual, so I'm hesitant to use the word "default". If the user does not affirmatively set a value then that secrete will not exist and our GitHub Workflows won't run. Instead I have re-phrased as:
 ```
`REGION` - Your choice of GCP region; quick start script suggests `us-central1`.
`ZONE`- Your choice of GCP zone; quick start script suggests `us-central1-a`.
```
After we implement the GitHub CLI in the quick start script to automate creation of the repository secrets it should be possible to have proper default values, values that are set for secrets if the user dose not specify a choice. At that point we can update our the language accordingly in this section.